### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by opatry**
While writing Kotlin bindings and associated tests, I noticed several typos in the documentation.

Here are some fixes proposal.

### CODE REVIEW

File: content/en/entities/featuredtag.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ menu:
 {
   "id": "627",
   "name": "nowplaying",
+  "url": "https://foo.bar",
```
Comment (opatry) [inline content/en/entities/featuredtag.md:15]: If you have a better URL to suggest here (or maybe the field isn't mandatory and then the field description would need an update).

**Comment by trwnh** [comment]
a lot of this is fixed or obsoleted by #991 but i'll check to see if there's anything new in here when i get home

**Comment by opatry** [comment]
I checked for redundant work in issues and PR and missed #991, sorry about that.
I was looking for documentation related issues/PR in title reason why I missed it.

I can rebase my branch on top of #991.

**Comment by opatry** [comment]
It appears that a lot of movement occurred in #991, file renaming etc. So might not be that immediate to simply rebase things.

Let me know if I can help anywhere.

**Pull request closed without merging**